Makefile

```diff
@@ -233,6 +233,11 @@ docker-build: ## Build docker image with the manager.
 	$(CONTAINER_TOOL) build -t ${IMG} -f Dockerfile.controller .
 	$(CONTAINER_TOOL) build -t ${IMG_DMST} -f Dockerfile.daemonset .
 
+.PHONY: docker-build-ocp
+docker-build-ocp: ## Build docker image with the manager.
```

Comment:
WDYT about renaming the target to `container-build-ocp` instead of `docker-build-ocp`?

Reply:
I wouldn't have requested for a change if this target is something thats related to `kind` or some other cluster(upstream). OCP seems to be less in tact with the usage of the term `docker` and hence I think we should remain consistent with it.